src/projects/project-overview-page.html

```diff
@@ -83,24 +86,91 @@
         height: 100%;
       }
 
-      feature-bar .content h2 {
-        margin-top: 0;
-      }
       feature-bar .content {
         display: flex;
+        justify-content: space-between;
       }
       feature-bar .content .title {
-        font-style: italic;
+        flex-grow: 1;
+        overflow: hidden;
+      }
+      feature-bar .content .title h2 {
+        margin: 0;
+        width: 100vw;
+      }
+      feature-bar .content #cloneUrl {
+        display: flex;
+        font-size: 14px;
+
+      }
+      feature-bar paper-button {
+        margin: 0;
+      }
+      feature-bar .clone-button {
+        background-color: var(--primary-action-green);
+        color: white;
+        font-weight: 600;
+      }
+      feature-bar .content #cloneUrl:not([closed]) .clone-button {
+        background-color: var(--primary-action-red);
+      }
+      feature-bar .content #cloneUrl .url-content {
+        line-height: 22px;
+        border-radius: 2px;
+        background-color: var(--secondary-background-color);
+        transition: width ease 0.25s;
+        width: 300px;
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
+        display: flex;
+        overflow: hidden;
+      }
+      feature-bar .content #cloneUrl .clone-url-input {
+        font-family: 'Roboto Mono';
+        font-size: 14px;
+        background-color: var(--secondary-background-color);
+        border: none;
+        margin: 8px;
+        flex: 1;
+      }
+      feature-bar #cloneUrl #toggleUrlType {
+        padding: 0;
+      }
+
+      #toggleUrlType[ssh-url] > .https {
+        display: none;
+      }
+      #toggleUrlType:not([ssh-url]) > .ssh {
+        display: none;
+      }
+
+      feature-bar .content #cloneUrl[closed] .url-content {
+        width: 0px;
+        padding: 0;
+      }
+      feature-bar .content .clone {
+        -webkit-box-shadow: -20px 0 20px 0px rgb(255, 255, 255);
+        -moz-box-shadow: -20px 0 20px 0px rgb(255, 255, 255);
+        box-shadow: -20px 0 20px 0px rgb(255, 255, 255);
       }
     </style>
 
     <app-route route="{{route}}" pattern="/:page" data="{{page}}"></app-route>
 
     <feature-bar selected-feature={{page.page}}>
       <div class="content">
-        <h2 class="title">[[project.owner]] / [[project.name]]</h2>
-        <span class="description">[[project._actual.description]]</span>
```

Comment:
I see you deleted the project description, was this a deliberate decision?
We might want to add this again to the header bar.